src/modules/comment/CommentCount.test.js

```diff
@@ -0,0 +1,33 @@
+const CommentCount = require('./CommentCount.js');
+
+describe('CommentCount', () => {
+  beforeEach(() => {
+    document.body.innerHTML = `
+      <div id="items">
+        <div class="comment">Comment 1</div>
+        <div class="comment">Comment 2</div>
+      </div>
+    `;
+  });
+
+  afterEach(() => {
+    document.body.innerHTML = '';
+  });
+
+  test('returns the correct number of comments', () => {
+    const count = CommentCount();
+    expect(count).toBe(3);
+  });
+
+  test('returns 0 if there are no comments', () => {
+    document.body.innerHTML = '<div id="items"></div>';
+    const count = CommentCount();
+    expect(count).toBe(0);
+  });
+
+  test('returns NaN if the total comments element is not found', () => {
+    document.body.innerHTML = '<div></div>';
+    const count = CommentCount();
+    expect(Number.isNaN(count)).toBe(true);
+  });
+});
```

Comment:
- [ ] As stated in the project requirement These count functions need to be covered with unit tests using Jest and should be passed , So kindly make sure that All test are passing.
![Capture](https://user-images.githubusercontent.com/93598189/221385432-482def11-bf86-4c50-b603-e1b4dda20529.PNG)